src/user-guide/task-implementation.rst

```diff
@@ -95,23 +95,22 @@ the ``$PATH`` in your bash configuration files (e.g. ``.bashrc``).
 Task Messages
 -------------
 
-Task jobs send status messages back to the server program to report that
+Task jobs send status messages back to the :term:`scheduler` to report that
```

Comment:
Am I correct that server program should be renamed to scheduler?

Reply:
You are indeed!